src/plugins/walletConnectToDapps/components/DappRegistryGrid.tsx

```diff
@@ -0,0 +1,96 @@
+import { SearchIcon } from '@chakra-ui/icons'
+import {
+  Box,
+  Heading,
+  Image,
+  Input,
+  InputGroup,
+  InputLeftElement,
+  Link,
+  SimpleGrid,
+  Stack,
+  Text as PlainText
+} from '@chakra-ui/react'
+import { Text } from 'components/Text'
+import { FC, useEffect, useMemo } from 'react'
+import { useForm } from 'react-hook-form'
+
+interface RegistryItem {
+  category: string
+  id: string
+  homepage: string
+  name: string
+  image: string
+}
+
+const registry: RegistryItem[] = require('../registry.json')
+
+const pageSize = 20
+
+export const DappRegistryGrid: FC = () => {
+  const { register, watch, setValue } = useForm<{ search: string; page: number }>({
+    mode: 'onChange',
+    defaultValues: {
+      search: '',
+      page: 0,
+    },
+  })
+
+  const search = watch('search')
+  const page = watch('page')
+  useEffect(() => setValue('page', 0), [search, setValue])
+
+  const filteredListings = useMemo(
+    () => registry.filter(l => !search || l.name.toLowerCase().includes(search.toLowerCase())),
+    [search],
+  )
+
+  return (
+    <Box>
+      <Stack direction='row' alignItems='center' mb={4}>
+        <Heading flex={1} fontSize='2xl'>
+          <Text translation='plugins.walletConnectToDapps.availableDapps' />
+        </Heading>
+        <Box>
+          <InputGroup>
+            <InputLeftElement pointerEvents='none'>
+              <SearchIcon color='gray.700' />
+            </InputLeftElement>
+            <Input
+              {...register('search')}
+              type='text'
+              placeholder='Search'
+              pl={10}
+              variant='filled'
+            />
```

Comment:
@shapeshift/product this has autocomplete from the asset search input, do we want to turn autocomplete off for it?

<img width="280" alt="image" src="https://user-images.githubusercontent.com/17035424/189231665-6c8c5ba1-3462-45de-a7ac-3d367a6425dd.png">

Reply:
yah turn off auto complete for this input please.

Reply:
Disabled autocomplete